frontend/src/pages/AccountSend.vue

```diff
@@ -179,6 +180,7 @@ import useWalletStore from 'src/store/wallet';
 // --- Other ---
 import { txNotify } from 'src/utils/alerts';
 import { BigNumber, Contract, getAddress, MaxUint256, parseUnits, formatUnits, Zero } from 'src/utils/ethers';
+import { Wallet } from 'ethers';
```

Comment:
We keep all ethers imports in `src/utils/ethers` since I _think_ it tree-shakes better if you don't import from the root `ethers` package (but that might not be true/outdated), and it just makes it clearer to track the set of packages we're dependent on. All that to say let's move `Wallet` to be exported in `src/utils/ethers` and import it from there
```suggestion
import { BigNumber, Contract, getAddress, MaxUint256, parseUnits, formatUnits, Zero, Wallet } from 'src/utils/ethers';
```

I think `export { Wallet } from '@ethersproject/wallet';` is all you need to add that that file

Reply:
Going to just remove the Wallet import in favor of the `RandomNumber` approach below